records_mover/db/connect.py

```diff
@@ -14,13 +14,38 @@
 # cred-service/LastPass/etc) into which driver to use.
 db_driver_for_type = {
     'postgres': 'postgresql',
+    'mysql': 'mysql+pymysql',
     'vertica': 'vertica+vertica_python',
 }
 
 odbc_driver_for_type = {
     'vertica': 'vertica+pyodbc',
 }
 
+query_for_type = {
+    'mysql': {
+        # This allows use of MySQL's bulk load mechanism--please see
+        # the URL below for security aspects of this.  We are assuming
+        # that the server is known and trusted, and that the network
+        # connection to it can be trusted.
+        #
+        # If the server is trusted but the network is not, we
+        # recommend you investigate the following:
+        #
+        # * Enabling SSL on your MySQL server.
+        # * Using the 'REQUIRE SSL' clause in the 'GRANT ALL
+        #   PRIVILEGES' statement used to create users.
+        # * Using pymysql's syntax to point to the correct CA cert for
+        #   the MySQL server and require validation.
+        #
+        # PRs to make this configuration easier to use with
+        # records-mover are welcome.
+        #
+        # https://dev.mysql.com/doc/refman/8.0/en/load-data-local-security.html
```

Comment:
There's a couple of concerns here:

1. Can we put this somewhere more prominent?  Maybe SECURITY.md or something?  Open to ideas on specifics.
2. Right now to actually make this work without forking, someone would need to do something like:
    ```python
     records_mover.db.connect.query_for_type.mysql['ssl'] = { some_crazy: params }
     ```

     This is maybe not awful but doesn't really seem like an intended interface.  Maybe this is a backlog item to allow better configuration of connect parameters, and also a line item to consider as we document what's a supported public interface and what's subject to change.

Reply:
I'm moving forward with the PR with the following:

1. Create SECURITY.md and see if there's a straightforward place to point to SECURITY.md from README.md
2. Create backlog item